### ISSUE TITLE
paramsSerializerOptions: arrayFormat does not seems to work.

### ISSUE THREAD

**Issue by omarkhatibco** [orval-labs/orval#1751]
Hello, 

I'm trying to use `arrayFormat ` but it does not seems to work, as in the docs 

```ts
paramsSerializerOptions: {
    qs: {
        arrayFormat: 'brackets',
    },
},
```

I'm not sure it's a bug since there is no qs library installed anyway. does this option only works if axios is used?

**Update:** I just tried to provide a custom params serializer function, it's not used in the generated code and all the code are using  `URLSearchParams`

**Comment by melloware** [comment]
I think we need much better documentation: https://orval.dev/reference/configuration/output#paramsserializeroptions

I have no idea what `qs` means or what the options for arrayFormat are!

**Comment by melloware** [comment]
@AllieJonsson these docs definitely need improving as I looked at it and it tells me absolutely nothing about this feature, what its for, what the values mean, Axios only etc?

**Comment by AllieJonsson** [comment]
I agree! I can try to both improve the docs and make this work with fetch, its quite a cool feature! 
`qs` is a npm package that stringifies query parameters from an object. E.g. 
```ts
{
  Foo: {
    Bar: "baz"
  }
}
```
Could become
`?Foo[Bar]=baz`

**Comment by soartec-lab** [comment]
@omarkhatibco @AllieJonsson 

`fetch` client now supports `explode` of `OpenAPI` with the following PR:

https://github.com/orval-labs/orval/pull/1604

This allows you to handle the serializer as defined by `OpenAPI` in the generator, rather than passing it as an option.
Although the behavior is different from `axios`, it is preferable to respect the `OpenAPI` definition.
If you want to customize other than automatic generation by definition, please use `customFetch`.

**Comment by soartec-lab** [comment]
As mentioned above, the issue can be resolved by using the `explode` option, so this issue will be closed.
